C++/C++.sublime-syntax

```diff
@@ -578,6 +582,155 @@ contexts:
                 - include: global
             - include: expressions
 
+  module:
+    # The export keyword can go before most anything.
+    - match: '\b(export)\s+'
+      captures:
+        1: storage.modifier.c++
```

Comment:
Both `export` and `import` should use the same group of keywords. The most appropriate choise might be `keyword.declaration.[export|import]`.

Reply:
I used `storage.modifier.c++` because that's what `extern` uses.  The `export` keyword is used to export any class, function, variable, etc, from a module and makes it usable for anybody `import`ing that module.  Would `keyword.declaration` be the most appropriate for that?  Is `storage.modifier.c++` even appropriate for `extern`?

Reply:
Many such keywords (`namespace` , `using`, ...) used to use `keyword.control` or `storage.[type|modifier]` in the past. Many of them have been rescoped to `keyword.declaration` to separate them from normal flow-control keywords and from data types. It appears C/C++ haven't been touched with regards to such changes as there are long lasting ideas/plans to completely rewrite the syntax using latest ST syntax engine features.

So the scope name may not be primary priority at this point.

As `namespace` or `using` are scoped `keyword.control` only, it may even be suitable to use that scope for `export/import` as well even though it is not state of the art. Whatever class is used, I'd vote for using the same group of scopes for both keywords.

Scopes may need a general overhaul anyway.

Reply:
I've switched `export`, `import`, and `module` to use `keyword.control.c++` to match how the current language syntax scopes keywords.